src/selectors/wallet$.js

```diff
@@ -12,30 +12,39 @@ export const activeWalletId$ = state => state.wallets.activeWalletId;
 
 const accountsArray$ = state => [...state.accounts.entries.values()];
 
+export const legacyWallet$ = createSelector(
+    legacyAccounts$,
+    (accounts) => accounts.length > 0 ? ({
+        id: LEGACY,
+        balance: accounts.reduce((sum, account) => sum + (account.balance || 0) * 1e5, 0),
+        accounts,
+        label: LEGACY_LABEL,
+        type: WalletType.LEGACY,
+        numberAccounts: accounts.length,
+    }) : undefined 
```

Comment:
So are we not tracking `wordsExported` for legacy wallets then?

Reply:
Only case where this would be relevant are contributor accounts whose owners accidentally skipped words. (And still don't have them.) Maybe that's not worth the effort .

Reply:
Ok, fair enough. The export function is still there.